src/reactable.jsx

```diff
@@ -344,6 +344,7 @@
             var Ths = [];
             for (var index = 0; index < this.props.columns.length; index++) {
                 var column = this.props.columns[index];
+                var thClass = column.key
```

Comment:
Agreed :+1: . Maybe this is better: `'reactable-th-' + column.key.toLowerCase()` because our column key may have UpperCase character.